src/checks_test.cc

```diff
@@ -306,5 +306,52 @@ TEST(LinterTest, CheckImports) {
             "Imports should specify the type 'MODULE' or 'PROTO'.");
 }
 
+TEST(LinterTest, CheckExpressionParantheses) {
+  LinterOptions option;
+
+  EXPECT_EQ(CheckExpressionParantheses(
+                "SELECT D where a AND b*2 OR (c AND x OR T);", option)
+                .GetErrors()
+                .size(),
+            2);
+
+  EXPECT_TRUE(
+      CheckExpressionParantheses("SELECT D where a or b or c;", option).ok());
+
+  EXPECT_TRUE(
+      CheckExpressionParantheses("SELECT D where a and b and c;", option).ok());
+  EXPECT_FALSE(
+      CheckExpressionParantheses("SELECT D where a and b or c;", option).ok());
+  EXPECT_FALSE(
+      CheckExpressionParantheses("SELECT D where a or b and c;", option).ok());
+  EXPECT_TRUE(CheckExpressionParantheses(
+                  "SELECT D where (       a or b   ) and c;", option)
+                  .ok());
+}
+
+TEST(LinterTest, CheckCountStar) {
+  LinterOptions option;
```

Comment:
Add few more tests here:
```
SELECT count ( 1 )
/* count(1) */ SELECT COUNT(*) -- count 1
```

Reply:
I added, along with inside string test.

Reply:
I meant two separate EXPECT_* :D

Reply:
Oh.. anyway, It is stronger this way :D
Also added seperate for smaller one.